Formula/lib/libmapper.rb

```diff
@@ -0,0 +1,45 @@
+class Libmapper < Formula
+  desc "Distributed system for media control mapping"
+  homepage "http://www.libmapper.org"
+  url "https://github.com/libmapper/libmapper/releases/download/2.4.2/libmapper-2.4.2.tar.gz"
+  sha256 "130d0761cd6006c2f7458771ae0484d53d601d85dba9315cdffd4da4c30bf86b"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "liblo"
+
+  uses_from_macos "zlib"
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+    ]
+
+    if build.head?
+      system "./autogen.sh", *args
+    else
+      system "./configure", *args
+    end
+
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"mpr_version.c").write <<~EOS
+      #include <stdio.h>
+      #include "mapper/mapper.h"
+      int main() {
+        printf("%s", mpr_get_version());
+        return 0;
+      }
+    EOS
+    system ENV.cc, "mpr_version.c", "-I#{include}", "-L#{lib}", "-lmapper", "-o", "mpr_version"
```

Comment:
I think this was the original idea, given the dependency?
```suggestion
    flags = shell_output("pkg-config --cflags --libs mapper").chomp.split
    system ENV.cc, "mpr_version.c", *flags, "-o", "mpr_version"
```